### ISSUE TITLE
Not recognizing h3 as a version header

### ISSUE THREAD

**Issue by negebauer** [hypermodules/gh-release#89]
standard version is currently using h2 headers for major and minor versions and h3 for patch versions, as seen in this issue https://github.com/conventional-changelog/standard-version/issues/469

gh-release does not check h3 headers for versions

is it possible it does?

**Comment by ungoldman** [comment]
Hello @negebauer, I'm afraid this is a very old and bothersome issue.

---

TL:DR; we've tried to support `standard-version` in the past, they keep changing and don't respect changelog conventions, we don't support them anymore.

---

Many moons ago standard-version was using h1 tags for major and minor versions, and h2 tags for patches. This was agreed to be non-semantic for a variety of reasons (please read https://github.com/conventional-changelog/standard-version/issues/208 for more context), and it was agreed that we would modify the template.

After much deliberation, it was agreed that that `#` (h1) would represent the title of the changelog, `##` would represent all versions, and `###` would be used for content headers (features, fixes, etc) specific to a version. This was based on https://keepachangelog.com, a widely adopted changelog convention that preexists `conventional-changelog`, and which this library (which also preexists `conventional-changelog`) is also based on.

For purely stylistic reasons, the `conventional-changelog` folks wanted to add `<small>` tag to patches so that they would look smaller (https://github.com/conventional-changelog/conventional-changelog/pull/237/files#diff-d3d08bd6ff9ff7e4b95f6d7966fd440bR2-R8). I conceded and added that to the PR I wrote for them, and they accepted that PR (https://github.com/conventional-changelog/conventional-changelog/pull/237).

For reasons beyond my understanding that fix did not ship for over a year. Eventually it was said to have made it to a major version release (https://github.com/conventional-changelog/standard-version/issues/208#issuecomment-489468757).

Now your issue on my repo leads me to understand that they've either removed that convention or some other template in their very large repository has taken precedence.

I do not pretend to understand the process `standard-version` or `conventional-changelog` undertake, and I spent a significant amount of time working with them to get them to respect _changelog conventions_.  Despite a previous agreement and my best efforts they've dropped support for the originally agreed upon convention. At the risk of being a curmudgeon, I am not going to spend anymore of my time doing that, and this library will not change to accommodate their frequent changes. I'm sorry I can't be more helpful but I have reached the limits of the amount of free labor I'm willing to give away for this particular problem.

If you really need to use this library for publishing your changelog to github with `standard-version`, I suggest forking the project and publishing your own version.

Best of luck. 🥂
